### PR TITLE
Added hdf5-tools and setup.py changes needed to support execution of …

### PR DESCRIPTION
**Pull request by hokiegeek2**
…io_test.py in CI/CD workflow, enabled io_test.py #407

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
     - uses: actions/checkout@v2
     - name: Install dependencies
       run: |
-        apt-get update && apt-get install -y libhdf5-dev libzmq3-dev python3-pip
+        apt-get update && apt-get install -y libhdf5-dev libzmq3-dev python3-pip hdf5-tools
```
Comment (ronawho) [inline .github/workflows/CI.yml:28]: nit, but I might put `hdf5-tools` next to `libhdf5-dev` since they're logically related

**Comment by hokiegeek2** [comment]
@ronawho That's a good nit! I will make that change.

File: .github/workflows/CI.yml (hunk 1 above)
Comment (hokiegeek2) [reply .github/workflows/CI.yml:28]: agree, will make change

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -53,7 +53,7 @@ jobs:
...
```
Comment (ronawho) [inline .github/workflows/CI.yml:56]: Move this one too?

Comment (hokiegeek2) [reply .github/workflows/CI.yml]: Argh, missed that, fixed now

**Pull request merged**